state/mysql/mysql.go

```diff
@@ -171,6 +190,23 @@ func (m *MySQL) parseMetadata(md map[string]string) error {
 	}
 	m.connectionString = meta.ConnectionString
 
+	// Cleanup interval
+	m.cleanupInterval = meta.CleanupInterval
+	s, ok := md[cleanupIntervalKey]
+	if ok && s != "" {
```

Comment:
Nit:

```suggestion
	if s := md[cleanupIntervalKey]; s != "" {
```

> The reason why we have an `ok` as second argument is because reading a key that doesn't exist always returns the zero value for the type, which for a string is the empty string. This make it impossible to know if the value is defined but empty, or not present. In this case, we just care if there's a value and we treat undefined and defined but empty the same.

Reply:
Done